lib/index.js

```diff
@@ -22,6 +23,34 @@ controller.on('ambient', (bot, message) => {
       bot.whisper(message, whisperMessage);
     });
   }
+}));
 
-  // logger.info('Message received', JSON.stringify({ message }, null, 2));
-});
+controller.on('file_shared', ((bot, message) => {
```

Comment:
Should we filter this by whitelisted channels too?

Reply:
yes, that was in progress, now done